lib/formatters/table.js

```diff
@@ -0,0 +1,134 @@
+"use strict";
+
+var chalk,
+    table,
+    pluralize,
+    drawReport,
+    drawTable;
+
+chalk = require("chalk");
+table = require("table").default;
+pluralize = require("pluralize");
```

Comment:
For what it's worth, in the specific case of `pluralize`, I would love to see more use of it if it will help with eventual internationalization. But I'm not on the team so :smile: